### DISCUSSION TITLE
Add option to delete save data from FlxG.save

### DISCUSSION THREAD

**Discussion by aesthetaey** [FunkinCrew/Funkin#1679] [Ideas]
Is there actually anyway? or resetting what I did in options in fnf without clicking options.Because my cousin really had to go in options and controls and change the whole fnf controls?  Not only that when I try to change it back,  it goes on Kick Starter, up is down? HELP- ALSO WHEN I TRY  CLICK ON  STORY MODE I CLICK ENTER AND IT GOES ON OPTIONS

**Comment by TheHENOOB** [comment]
Go to: `C:\Users\[YOUR USERNAME]\AppData\Roaming\ninjamuffin99\Funkin\ninjamuffin99` folder and delete funkin.sol if you are on windows

**Comment by aesthetaey** [comment]
what if your on  browser?

**Comment by ghost** [comment]
go to funkin on newgrounds or itch.io and delete the cookies for uploads.ungrounded.net and itch.io

**Comment by aesthetaey** [comment]
but how do you do that -

**Comment by TheHENOOB** [comment]
> but how do you do that -

I recommend you to search "how to clear cookies or cache" for your browser

**Comment by ghost** [comment]
While on the page, click the lock. Then, click on something that shows cookies (it will say cookies somewhere)

Then, if on itch.io, select the websites one by one, pressing clear for each. If on Newgrounds, clear cookies for uploads.ungrounded.net

**Comment by aesthetaey** [comment]
thanks, it works!

**Comment by ninjamuffin99** [comment]
yeah thats current solution for now, eventually there will be an in-game thing to delete data and whatnot!

**Comment by libauror-a** [comment]
this shoud be easy using by using the FlxG.save.clear() method.
